Cython/Compiler/ModuleNode.py

```diff
@@ -3074,6 +3079,11 @@ def generate_module_init_func(self, imported_modules, env, code):
 
         # start of module init/exec function (pre/post PEP 489)
         code.putln("{")
+        
+        code.putln("#if !CYTHON_PEP489_MULTI_PHASE_INIT")
+        code.putln("if (__Pyx_VersionSanityCheck() < 0) return NULL;")
```

Comment:
This isn't C89 compatible. I'm assuming that's OK and that we're dropping that requirement.

Reply:
The C89 issue is that it comes before some variable definitions. That was mainly because some executable stuff (refnanny mainly) is mixed in with the variable definitions. I wanted to keep this check before anything else that could go wrong.

I'm returning `NULL` because it's the module init function. It returns a `PyObject*`